src/sentry/issues/escalating.py

```diff
@@ -154,36 +162,108 @@ def _query_with_pagination(
 ) -> List[GroupsCountResponse]:
     """Query Snuba for event counts for the given list of project ids and groups ids in
     a time range."""
+    organization = Organization.objects.get(id=organization_id)
+
     all_results = []
     offset = 0
+
     while True:
-        query = _generate_query(project_ids, group_ids, offset, start_date, end_date, category)
+        query = _generate_entity_dataset_query(
+            project_ids, group_ids, offset, start_date, end_date, category
+        )
         request = Request(
             dataset=_issue_category_dataset(category),
             app_id=REFERRER,
             query=query,
             tenant_ids={"referrer": REFERRER, "organization_id": organization_id},
         )
         results = raw_snql_query(request, referrer=REFERRER)["data"]
+
         all_results += results
         offset += ELEMENTS_PER_SNUBA_PAGE
         if not results or len(results) < ELEMENTS_PER_SNUBA_PAGE:
             break
 
+    if category == GroupCategory.ERROR and features.has(
+        "organizations:escalating-issues-v2", organization
+    ):
+        # Generate and execute the query to the Generics Metrics Backend
+        metrics_query = _generate_generic_metrics_backend_query(
+            organization_id, project_ids, group_ids, start_date, end_date, category
+        )
+        projects = Project.objects.filter(id_in=project_ids)
+        metrics_results = get_series(
+            projects=projects, metrics_query=metrics_query, use_case_id=UseCaseID.ESCALATING_ISSUES
+        )
+
+        # Log exception if results from the Metrics backend are
+        # not equivalent to the Errors dataset.
+        if metrics_query != all_results:
+            logger.info(
+                "Generics Metrics Backend query results not the same as Errors dataset query.",
+                extra={"metrics_results": metrics_results, "dataset_results": all_results},
+            )
+
     return all_results
 
 
-def _generate_query(
+def _generate_generic_metrics_backend_query(
+    organization_id: int,
+    project_ids: Sequence[int],
+    group_ids: Sequence[int],
+    start_date: datetime,
+    end_date: datetime,
+    category: Optional[GroupCategory],
+):
+    """
+    This function generates a query to fetch the hourly events
+    for a group_id through the Generic Metrics Backend.
+
+    The Generic Metrics Backend only contains data for Errors.
+    """
+
+    # Check if category is for Errors, else raise an exception
+    if category is None or category != GroupCategory.ERROR:
+        raise Exception("Invalid category.")
+
+    select = [
+        MetricField(metric_mri=ErrorsMRI.EVENT_INGESTED.value, alias="event_ingested", op=None),
```

Comment:
In this field you didn't specify an `op` and I see you didn't define neither a `DerivedOp` nor a `SingularEntityDerivedMetric`, thus [this check](https://github.com/getsentry/sentry/blob/24148fcfe0dc7033c59bf28f574203dd4250ae37/src/sentry/snuba/metrics/fields/base.py#L1836-L1836) will fail.

Which aggregation operation would you like to use?